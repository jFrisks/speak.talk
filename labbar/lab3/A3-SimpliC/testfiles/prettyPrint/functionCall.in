int b() {
    int x;
    x = 10;
}
int a() {
    int x;
    int y;
    x = 10;
    y = 11;
    print(b(x, y));
}