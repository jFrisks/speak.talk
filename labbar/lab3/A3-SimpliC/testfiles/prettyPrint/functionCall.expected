int b(){
    int x;
    x = 10;
}
int a(){
    int x;
    x = 10;
    int y = 11;
    print(b(x, y));
}